styles.css

```diff
@@ -1,11 +1,15 @@
-@import url('./Components/elixir.css');
+@import url('/Components/elixir.css');
 
 /* grid-container */
+:root {
+    --navbar-height: 10vh;
+}
 
 .grid-container {
     display: grid;
-    grid-template-columns: 250px 1fr 1fr 1fr 1fr 1fr;
+    grid-template-columns: auto 1fr 1fr 1fr 1fr 1fr;
```

Comment:
here you can use repeat(5, 1fr).

Reply:
Yes!
Thank you for your time and review! 👍🏽